dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditReplayThread.java

```diff
@@ -254,7 +258,16 @@ public FileSystem run() {
           fs.concat(new Path(src), dsts.toArray(new Path[] {}));
           break;
       }
+
+      String key = command.getSimpleUgi() + "_" + replayCommand.getType().toString();
```

Comment:
Using a String concatenation doesn't need like the most efficient or clean route. I think it should be pretty easy to create a composite key like `class UserCommandKey implements Writable` which has two `Text` fields that it delegates to for serialization.